internal/driver/controllerserver_helper.go

```diff
@@ -440,6 +481,10 @@ func (cs *ControllerServer) createVolumeContext(ctx context.Context, req *csi.Cr
 		volumeContext[LuksKeySizeAttribute] = req.GetParameters()[LuksKeySizeAttribute]
 	}
 
+	if req.GetParameters()[VolumeEncryption] == True {
+		volumeContext[VolumeEncryption] = True
+	}
+
```

Comment:
question: are we referencing or need to use it during other steps of PVC lifecycle? Not sure if we need to set this

Reply:
I have set it for future purposes, just in case there is a need for it.